infra/scripts/v2t-workstation.sh

```
@@ -0,0 +1,596 @@
+  wait_for_container_health graphiti-mcp-graphiti-mcp-1
+
+  require_single_line_value "REPO_ROOT" "$REPO_ROOT"
+  require_single_line_value "GRAPHITI_PROXY_SERVICE_NAME" "$GRAPHITI_PROXY_SERVICE_NAME"
+
+  run_as_target_user env \
+    REPO_ROOT="$REPO_ROOT" \
+    GRAPHITI_PROXY_SERVICE_NAME="$GRAPHITI_PROXY_SERVICE_NAME" \
+    bash -lc '
+      cd "$REPO_ROOT"
+      GRAPHITI_PROXY_SERVICE_NAME="$GRAPHITI_PROXY_SERVICE_NAME" bash scripts/graphiti-proxy-service-install.sh
+    '
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)